### PR TITLE
feat: banana coin

### PR DESCRIPTION
**Pull request by Scale0**
a redo of Murtagh's PR for the banana coin

### CODE REVIEW

File: tokens/main/0xf01069227b814f425bad4ba70ca30580f2297ae8/additional.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "website": "Soon",
```
Comment (XJWX89) [inline tokens/main/0xf01069227b814f425bad4ba70ca30580f2297ae8/additional.json:2]: I suggest removing the `website` for now, and adding it when the site is ready.

File: tokens/test/0xca92f3d24c3f04c3ab7711f5a8981b48900d1baf/info.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Banana Coin",
```
Comment (XJWX89) [inline tokens/test/0xca92f3d24c3f04c3ab7711f5a8981b48900d1baf/info.json:2]: After checking the smart contract, it should be `BananaCoin`

File: tokens/test/0xca92f3d24c3f04c3ab7711f5a8981b48900d1baf/additional.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "website": "Soon",
```
Comment (XJWX89) [inline tokens/test/0xca92f3d24c3f04c3ab7711f5a8981b48900d1baf/additional.json:2]: I suggest removing the website for now, and adding it when the site is ready.

File: tokens/main/0xf01069227b814f425bad4ba70ca30580f2297ae8/info.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Banana Coin",
```
Comment (XJWX89) [inline tokens/main/0xf01069227b814f425bad4ba70ca30580f2297ae8/info.json:2]: After checking the smart contract, it should be `BananaCoin`

File: tokens/main/0xf01069227b814f425bad4ba70ca30580f2297ae8/additional.json (hunk 1 above)
Comment (Scale0) [reply tokens/main/0xf01069227b814f425bad4ba70ca30580f2297ae8/additional.json]: fixxed: removed website for now

File: tokens/test/0xca92f3d24c3f04c3ab7711f5a8981b48900d1baf/additional.json (hunk 3 above)
Comment (Scale0) [reply tokens/test/0xca92f3d24c3f04c3ab7711f5a8981b48900d1baf/additional.json]: fixxed: removed website for now

File: tokens/test/0xca92f3d24c3f04c3ab7711f5a8981b48900d1baf/info.json (hunk 2 above)
Comment (Scale0) [reply tokens/test/0xca92f3d24c3f04c3ab7711f5a8981b48900d1baf/info.json]: fixxed: changed coin name to BanaCoin

File: tokens/main/0xf01069227b814f425bad4ba70ca30580f2297ae8/info.json (hunk 4 above)
Comment (Scale0) [reply tokens/main/0xf01069227b814f425bad4ba70ca30580f2297ae8/info.json]: fixxed: changed coin name to BanaCoin

**Review by XJWX89 — APPROVED** [review]

**Pull request merged**